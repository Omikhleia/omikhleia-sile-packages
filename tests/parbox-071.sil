\begin{document}
\neverindent
\nofolios
\script[src=packages/parbox]
\script[src=packages/lorem]
% Test case 071 = As 070, regular vs. minimized, but we need another test for
% padding, both for the regular and the minimized parbox.

(7A) \parbox[width=37%fw, valign=middle, border=0.5pt, padding=5pt]{First\par
37\%fw width\par
Not minimized}

\smallskip

(7B) \parbox[width=37%fw, valign=middle, border=0.5pt, padding=5pt, minimize=true]{Second\par
37\%fw width\par
Minimized}

\end{document}