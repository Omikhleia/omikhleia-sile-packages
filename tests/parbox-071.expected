Set paper size 	595.275597	841.8897729
Begin page
Mx 	29.7638
My 	62.4783
Set font 	Gentium Plus;10;400;;normal;;LTR
T	11 w=3.1689	(()
Mx 	32.9327
T	26 36 w=10.6445	(7A)
Mx 	43.5773
T	12 w=3.1689	())
Draw line	49.4227	42.0945	208.2268	0.5000
Draw line	49.4227	82.8621	208.2268	0.5000
Draw line	49.4227	42.0945	0.5000	41.2676
Draw line	257.6495	42.0945	0.5000	41.2676
Mx 	54.4227
My 	53.7156
T	41 76 85 86 87 w=18.7549	(First)
My 	65.7156
T	22 26 w=9.3848	(37)
Mx 	63.8074
T	8 w=6.8994	(%)
Mx 	70.7069
T	73 90 w=10.1709	(fw)
Mx 	83.5363
T	90 76 71 87 75 w=23.9307	(width)
Mx 	54.4227
My 	77.7156
T	49 82 87 w=14.9609	(Not)
Mx 	72.0413
T	80 76 81 76 80 76 93 72 71 w=43.7744	(minimized)
Mx 	29.7638
My 	107.7195
T	11 w=3.1689	(()
Mx 	32.9327
T	26 37 w=10.1904	(7B)
Mx 	43.1232
T	12 w=3.1689	())
Draw line	48.9846	86.8621	63.0283	0.5000
Draw line	48.9846	128.5769	63.0283	0.5000
Draw line	48.9846	86.8621	0.5000	42.2148
Draw line	112.0129	86.8621	0.5000	42.2148
Mx 	53.9846
My 	99.4304
T	54 72 70 82 81 71 w=29.4971	(Second)
My 	111.4304
T	22 26 w=9.3848	(37)
Mx 	63.3694
T	8 w=6.8994	(%)
Mx 	70.2688
T	73 90 w=10.1709	(fw)
Mx 	83.0823
T	90 76 71 87 75 w=23.9307	(width)
Mx 	53.9846
My 	123.4304
T	48 76 81 76 80 76 93 72 71 w=43.8037	(Minimized)
End page
Finish
